heartbeat/gcp-vpc-move-route.in

```diff
@@ -68,7 +68,7 @@ metadata() {
 cat <<END
 <?xml version="1.0"?>
 <!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
-<resource-agent name="gcp-vpc-move-ip">
+<resource-agent name="gcp-vpc-move-route">
```

Comment:
Can you undo this change and rename your Python gcp-vpc-move-ip to gcp-vpc-move-route.in ? This way it will auto-generate when you run ./autogen.sh; ./configure

I'll add some Python detection code, so we can set specific path's (e.g. for python2 or 3) instead of using env.

Reply:
sure, I'll update the PR.